unit/solvers/smt2_incremental/encoding/enum_encoding.cpp

```diff
@@ -28,50 +29,61 @@ static c_enum_typet make_c_enum_type(const typet &underlying_type)
   return enum_type;
 }
 
-TEST_CASE("enum encoding of expressions", "[core][smt2_incremental]")
+TEST_CASE("enum corner cases", "[core][smt2_incremental]")
```

Comment:
I preferred the previous test description, as maybe we shouldn't label the entirety of the enum tests as corner cases.